src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -91,7 +91,11 @@ def execute(self, unexpanded_argv):  # pylint: disable=too-many-statements
             enable_autocomplete(self.parser)
             az_subparser = self.parser.subparsers[tuple()]
             _help.show_welcome(az_subparser)
-            telemetry.log_telemetry('welcome')
+
+            # TODO: Question, is this needed?
+            telemetry.set_command_details('az')
+            telemetry.set_success(summary='welcome')
```

Comment:
The purpose was to track the use of just running `$ az`.
I'd still see value in keeping that.